api/v1/zz_generated.conversion.go

```diff
@@ -138,11 +138,6 @@ func autoConvert_unversioned_Image_To_v1_Image(in *unversioned.Image, out *Image
 	return nil
 }
 
-// Convert_unversioned_Image_To_v1_Image is an autogenerated conversion function.
-func Convert_unversioned_Image_To_v1_Image(in *unversioned.Image, out *Image, s conversion.Scope) error {
-	return autoConvert_unversioned_Image_To_v1_Image(in, out, s)
-}
-
```

Comment:
will look into why this got removed, i _did_ hit some issues when I last generated code due to not being able to covert `unversioned...Pinned` to `<version>...Pinned` as the `Pinned` field did/does not exist in versioned generations.